core/domain/exp_jobs_one_off.py

```diff
@@ -433,6 +433,58 @@ def reduce(key, values):
             no_of_models_deleted)])
 
 
+class ExplorationRteMathContentValidationOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """Job that checks the html content of an exploration and validates the
+    Math content object for each math rich-text components.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_fetchers.get_exploration_from_model(item)
+        invalid_tags_info_in_exp = []
+        for state_name, state in exploration.states.items():
+            html_string = ''.join(state.get_all_html_content_strings())
+            error_list = (
+                html_validation_service.
+                validate_math_content_in_math_rich_text(html_string))
```

Comment:
optional: `validate_math_content_attribute_in_html` as the function param is an HTML!

Reply:
done